src/main/java/com/github/shiranr/scooters/domain/internal/History.java

```diff
@@ -0,0 +1,14 @@
+package com.github.shiranr.scooters.domain.internal;
+
+/**
+ * History of a scooter is composed from check in and out of the scooter by a user
+ */
+public class History {
```

Comment:
nit - possibly call this HistoryRecord, as a list of HistoryRecord creates the history? (not a must here, just a suggestion :))

Reply:
Will be done in next commit.